airbyte-integrations/connectors/destination-pinecone/metadata.yaml

```diff
@@ -9,10 +9,16 @@ data:
       enabled: true
     oss:
       enabled: true
+  resourceRequirements:
+    jobSpecific:
```

Comment:
you should leave a comment pointing to a ticket whose resolution would make this memory requirement unnecessary